### PR TITLE
Add several spelling corrections + refinements

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4412,8 +4412,11 @@ bultin->builtin
 bumb->bump
 bumpded->bumped
 bumpted->bumped
+bund->bound
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt:4415]: Also "bind"

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -5211,6 +5215,13 @@ chemestry->chemistry
 chemicaly->chemically
 chenged->changed
 chennel->channel
+cherch->church
+cherching->checking
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt:5219]: Also "churching"

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -5211,6 +5215,13 @@ chemestry->chemistry
...
+cherchs->churchs
+cherck->check
+chercked->checked
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt:5222]: Probably also "checkered", but further away

**Comment by luzpaz** [comment]
I don't understand why the CI is failing. Also what's the deal with the duplicate words are the end of `dictionary.txt`

https://github.com/codespell-project/codespell/blob/master/codespell_lib/data/dictionary.txt#L31985-L31992

**Comment by arm-in** [comment]
> Also what's the deal with the duplicate words are the end of `dictionary.txt` (exlcluding lines 31990 and 31991)

I think the lines at the end are Unicode.
U+0441 = HEX D1 81 = cyrillic "c"

**Comment by arm-in** [comment]
> > Also what's the deal with the duplicate words are the end of `dictionary.txt` (excluding lines 31990 and 31991)
> 
> I think the lines at the end are Unicode.
> U+0441 = HEX D1 81 = cyrillic "c"

When I think about it ...

The dictionaries of codespell are British English or American English, whatsoever.
There should be no cyrillic letters contained in *any* word.
Instead of adding lots of words in the dictionary with bogus letters, there should a filter in codespell to indicate / correct those.

**Comment by luzpaz** [comment]
Added requested revisions from @arm-in   
In regards to cyrillic character issue, postponing that for now.

**Pull request merged**